src/RoundFactory.sol

```diff
@@ -20,9 +20,12 @@ contract RoundFactory is IRoundFactory, Initializable, UUPSUpgradeable, Ownable
     /// @notice The v1 recurring round beacon contract address.
     address public immutable recurringRoundV1Beacon;
 
-    /// @notice Address authorized to sign `Claim` messages
+    /// @notice Address authorized to sign `Claim` messages.
     address public signer;
 
+    /// @notice Address authorized to distribute funds via a contract call.
+    address public distributor;
```

Comment:
Shifting storage - reset by setting fee claimer and fee BPS following deployment.